frontend/pages/LabelPage/LabelForm/LabelForm.tsx

```diff
@@ -58,21 +61,28 @@ const LabelForm = ({
   isEdit,
   onCancel,
   handleSubmit,
+  onOpenSchemaSidebar,
   onOsqueryTableSelect,
+  showOpenSchemaActionText,
   backendValidators,
 }: ILabelFormProps): JSX.Element => {
-  const [name, setName] = useState<string>(selectedLabel?.name || "");
+  const [name, setName] = useState<string>("");
   const [nameError, setNameError] = useState<string>("");
-  const [description, setDescription] = useState<string>(
-    selectedLabel?.description || ""
-  );
+  const [description, setDescription] = useState<string>("");
   const [descriptionError, setDescriptionError] = useState<string>("");
-  const [query, setQuery] = useState<string>(selectedLabel?.query || "");
+  const [query, setQuery] = useState<string>("");
   const [queryError, setQueryError] = useState<string>("");
   const [platform, setPlatform] = useState<string>(
     selectedLabel?.platform || ""
   );
 
+  useEffect(() => {
```

Comment:
Was able to fix this. It wasn't loading right away since I was rendering the component before the data was received from the API, added a loading state that fixed it